ui/lib/core/addon/components/page/breadcrumbs.hbs

```diff
@@ -0,0 +1,18 @@
+<nav class="breadcrumb" aria-label="breadcrumbs" data-test-breadcrumbs>
+  <ul>
+    {{#each @breadcrumbs as |breadcrumb|}}
+      <li>
+        <span class="sep">/</span>
+        {{#if breadcrumb.linkExternal}}
+          <LinkToExternal @route={{breadcrumb.path}}>{{breadcrumb.label}}</LinkToExternal>
```

Comment:
Small nit but would `route` be a better name for the breadcrumb prop than path in case someone would think that it expects `configuration/edit` rather than `configuration.edit`?

Reply:
yeah - I had copied `path` over, but I agree `route` makes more sense here